src/apel_plugin/publish.py

```diff
@@ -31,15 +32,16 @@
 
 
 async def run(config, client):
+    logging.critical("START6")
```

Comment:
Is this only for debugging or should this also be in production? If yes, maybe a more descriptive log message would be good.

Reply:
removed with latest commit